src/sql/src/func.rs

```diff
@@ -3847,7 +3847,7 @@ pub static OP_IMPLS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|| {
             params!(Char, String) => Operation::binary(|ecx, lhs, rhs| {
                 let length = ecx.scalar_type(&lhs).unwrap_char_length();
                 Ok(lhs.call_unary(UnaryFunc::PadChar(func::PadChar { length }))
-                    .call_binary(rhs, IsLikeMatch { case_insensitive: false })
```

Comment:
Wow! Uh, so this PR will do the opposite behavior for all installed views? We might need to look through customer views and inform any of these.

Reply:
Yes, ILIKE was almost completely broken due to all these bugs. I'll try to write a query that we can run against all customer environments to see if anyone was affected without them knowing.